instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -167,6 +175,18 @@ def _instrumented_requests_call(
             SpanAttributes.HTTP_URL: url,
         }
 
+        metric_labels = {
+            "http.method": method,
+            "http.url": url,
```

Comment:
Is this `url` parametrized or plain raw url? Otherwise, this can lead very to high cardinality.

Reply:
This is the same `url` we add in span attributes. I think its parameterised. (i.e. when added in metric it would look something like this `http.url: STRING(https://www.google.com?key1=value1`)

Reply:
That's not a problem with tracing but here each url creates a new time series which should be avoided. If there is no way to get the parametrized url I would suggest remove this tag entirely.

Reply:
Removed url tag